### ISSUE TITLE
Restructure configuration.toml for new Writeable section and Abstract Registry

### ISSUE THREAD

**Issue by lenny-intel** [edgexfoundry/device-modbus-go#22]
Once the Abstract Registry integration into device-sdk-go is merged the go.mod file needs to be updated to pull latest device-sdk-go and then the following changes need to be made to the following files:
```
    - cmd/res/configuration.toml
    - cmd/res/docker/configuration.toml
```
Add the Writable section as:

```
[Writable]
LogLevel = 'INFO'
```

Modify the Logging section to be:
```
[Logging]
EnableRemote = false
File = "./device-simple.log"
```

Also add following to Registry section
```
Type = "consul"
```

**Comment by lenny-intel** [comment]
sivakrishnan-intel will be making these changes.

**Comment by lenny-intel** [comment]
Addressed by PRs #28 & #29
